intro-HPC/title.tex

```diff
@@ -1,21 +1,15 @@
- \begin{center}
+\begin{center}
 
-\includegraphics*[width=8truecm]{logo_vsc}
+\includegraphics*[width=12truecm]{logo_vsc_2}
 
-\vspace*{1.5\baselineskip}
+\vspace*{6\baselineskip}
 
 \Huge \strong{HPC Tutorial} \\
-\LARGE Last updated: DATEPLACEHOLDER
+\LARGE Last updated: September 25 2018
```

Comment:
@sbecuwe Please keep `DATEPLACEHOLDER` here, this is automatically replaced by today's date when the PDF are generated by Travis.

Reply:
Done. So we don't have to run add-timestamp.sh any longer?